release: v1.52.0 (#5719)
apollographql/router: .changesets/fix_spawn_blocking_parser.md

> ### use spawn_blocking for query parsing & validation ([PR #5235](https://github.com/apollographql/router/pull/5235))
> 
> Moves query parsing and validation in a tokio blocking task to prevent all executor threads from blocking on large queries.
> 

Reviewer: ```suggestion
### Use spawn_blocking for query parsing and validation ([PR #5235](https://github.com/apollographql/router/pull/5235))

To prevent its executor threads from blocking on large queries, the router now runs query parsing and validation in a Tokio blocking task.

```
